### DISCUSSION TITLE
A couple different things that should possibly be changed or added

### DISCUSSION THREAD

**Discussion by DantheChef** [tbrittain/SmbExplorerCompanion#119] [Ideas]
Just for reference, I'm not really sure which of these are bugs and which are just not implemented yet.

- Make it so for both Career Batting/Pitching and Season Batting/Pitching, you're able to sort in ascending order as well as descending. Both the teams page and the teams season page currently have the ability to do both but not the other stat pages.
- Add the stadium that each team plays at somewhere, either on the Teams page or the teams actual page or both, that would just be a cool thing to have.
- Another cool thing would be a scale factor for games, I play 84 games a season but I would love to just see the stats scaled to 162 games or even down to 32 games just to see.

**Comment by tbrittain** [comment]
Your first point is planned for the next release: #50

I love the idea of keeping track of the stadium! I will probably handle it in a way that allows you to manage which stadium a team played their home games at on a per season basis.
And a scaling factor is a great idea too. I am trying to think about how best to handle this, but it will likely be a configurable option that you can set somewhere. I'm not quite sure how I'd want to display the scaled stats, but maybe something like a toggle on the stat grids to view stats/162 where applicable? I'd normally say that I'll just add the columns to the grid and only show them when you have a scaling factor in place, but I have had some issues conditionally displaying grid columns. I'll keep looking into this though. I'll make some backlog items to track these soon

**Reply by DantheChef** [reply]
Yeah, stadiums are just a nice to have thing, it's really not required at all but it's a bit of flavor which I always find fun. Just because I was thinking about it, and because I saw it [here](https://github.com/tbrittain/SmbExplorerCompanion/issues/52#issuecomment-1679006797), stuff like team/player bios, rivals, those kinds of things to just flavor things. It could be just a little text box to put things in really, nothing fancy but just a nice to have thing, not at all required though.

I'm not too into it but stadium adjusted stats but they could also be a thing if you would want them. WRC+ is one but I think OPS+ works just fine as it is, they're always within a few points of each other anyway.

For a scaling factor, maybe I'm just misreading this but, just a straight scaling factor would be better than a per 162 one, similar to per 48 like they use in basketball. So if for example, I played a 32 game season, and the player played 16 of those games, instead of scaling the stats to 162, as a per 162 stat would do, it would scale them to 81 games.
